commcare_connect/opportunity/models.py

```diff
@@ -70,3 +70,18 @@ class DeliverForm(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class OpportunityAccess(models.Model):
+    user = models.OneToOneField(
+        ConnectUser,
+        on_delete=models.CASCADE,
+        related_name="claims",
```

Comment:
I dont think you want to call these claims (default is probably fine, to be honest), since you can have access without yet having claimed the opportunity. In other words, some of these rows represent claims but some do not, so naming them like that is confusing.

Reply:
Fixed. [4ad1eaf](https://github.com/dimagi/commcare-connect/pull/41/commits/4ad1eaf0cda305b5fbe0d1cde7ecfd476fcbbf52)